
# HG changeset patch
# User bchristi
# Date 1538777145 25200
# Node ID 8513ac27b65198d2e6562fb7da6e9c99b9fdcf51
# Parent  a3b6f77c5378ba061f3c9ae036cafc1933952b92
8211731: Reconsider default option for ClassPathURLCheck change done in JDK-8195874
Reviewed-by: alanb, mchung

diff -r a3b6f77c5378 -r 8513ac27b651 src/java.base/share/classes/jdk/internal/loader/URLClassPath.java
--- a/src/java.base/share/classes/jdk/internal/loader/URLClassPath.java	Thu Oct 04 02:19:13 2018 -0700
+++ b/src/java.base/share/classes/jdk/internal/loader/URLClassPath.java	Fri Oct 05 15:05:45 2018 -0700
@@ -103,7 +103,7 @@
         DISABLE_ACC_CHECKING = p != null ? p.equals("true") || p.equals("") : false;
 
         // This property will be removed in a later release
-        p = props.getProperty("jdk.net.URLClassPath.disableClassPathURLCheck");
+        p = props.getProperty("jdk.net.URLClassPath.disableClassPathURLCheck", "true");
 
         DISABLE_CP_URL_CHECK = p != null ? p.equals("true") || p.isEmpty() : false;
         DEBUG_CP_URL_CHECK = "debug".equals(p);

